File: obj-c/obj-c/PeachesViewController.m
#import "PeachesViewController.h"

@interface PeachesViewController ()

@property (weak, nonatomic) IBOutlet UILabel *fruitAmount;
@property (weak, nonatomic) IBOutlet UITextView *peachesDlTextView;

@end

@implementation PeachesViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
//
    if (self.deepLinkData != nil) {
        self.peachesDlTextView.attributedText = [self attributionDataToString:self.deepLinkData];
        self.peachesDlTextView.textColor = UIColor.labelColor;
        self.fruitAmount.text = [self getFruitAmount:self.deepLinkData];
    }
}

- (IBAction)copyShareInviteLink:(UIButton *)sender {
    [super copyShareInviteLinkWithFruitName:@"peaches"];
}

@end

File: swift/basic_app/basic_app/PeachesViewController.swift
//
//  BananasViewController.swift
//  basic_app
//
//  Created by Liaz Kamper on 01/06/2020.
//  Copyright © 2020 OneLink. All rights reserved.
//

import UIKit

class PeachesViewController: DLViewController {
    
    @IBOutlet weak var fruitAmount: UILabel!
    @IBOutlet weak var peachesDlTextView: UITextView!

    override func viewDidLoad() {
        super.viewDidLoad()

        if (deepLinkData != nil) {
            peachesDlTextView.attributedText = attributionDataToString(data: deepLinkData!)
            peachesDlTextView.textColor = .label
            fruitAmount.text = getFruitAmount(data: deepLinkData!)
        }
    }
    
    @IBAction func copyShareInviteLink(_ sender: UIButton) {
        super.copyShareInviteLink(fruitName: "peaches")
    }
}

